cliquet/utils.py

```diff
@@ -9,12 +9,19 @@
 from binascii import hexlify
 from six.moves.urllib import parse as urlparse
 
-# ujson is not installable with pypy
+# ultrajson is not installable with pypy
 try:
-    import ujson as json  # NOQA
+    import ultrajson as json  # NOQA
+
+    def json_serializer(v, **kw):
+        return json.dumps(v, escape_forward_slashes=False)
```

Comment:
nit: I'm not 100% sure that it is better, but you could do `json_serializer = functools.partial(json.dumps, escape_forward_slashes=False)`